js_modules/dagster-ui/packages/ui-core/src/ui/Sidebar/SidebarDisclosureTriangle.tsx

```diff
@@ -0,0 +1,28 @@
+import {Icon, UnstyledButton} from '@dagster-io/ui-components';
+
+export const SidebarDisclosureTriangle = ({
+  isOpen,
+  toggleOpen,
+}: {
+  isOpen: boolean;
+  toggleOpen: () => void;
+}) => (
+  <UnstyledButton
+    onClick={(e) => {
+      e.stopPropagation();
+      toggleOpen();
+    }}
+    onDoubleClick={(e) => {
+      e.stopPropagation();
+    }}
```

Comment:
The `stopPropagation` on these is a bit of a code smell, indicating that we've got some nesting of interactive elements that we should avoid. It looks like this is a `button` rendered inside of a `Box` with a click handler, is that right? Can we avoid having this button be a child of an interactive element, perhaps as an absolutely positioned sibling of the `Box` (which should itself probably be a `button` too) within a shared relative positioned container?